<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Robotiq_2FingerPluginWidget</class>
 <widget class="QWidget" name="Robotiq_2FingerPluginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>690</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Control</string>
    </attribute>
    <widget class="QSlider" name="slider_position">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pbn_gripperClose">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbn_gripperOpen">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>100</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_position">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>100</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Position :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Force :</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_force">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_force">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>160</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Speed :</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_speed">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>241</width>
       <height>29</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_speed">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>230</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbn_gripperAction">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>330</y>
       <width>99</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Action</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>691</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string>Port Settings:</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Port :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx_port">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Baudrate :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbx_baudrate"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Data bits:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbx_dataBits"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Stop Bits:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbx_stopBits"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Parity :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cbx_parity"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Flow control  :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbx_flowControl"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pbn_connect">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbn_disableGripper">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbn_activeGripper">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>380</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Active Gripper</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>481</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Info:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>481</y>
       <width>332</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>While booting, status LED will be solid blue / red.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>520</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>520</y>
       <width>332</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gripper is not online.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>About </string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>741</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Description :</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
